src/main/scala/cromwell/authentication/DockerCredentials.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.authentication
```

Comment:
Yes initially I intended to factorize more stuff than that but it ended up being more tightly coupled than I thought. I still think in the future if we end up supporting other backends with different kind of authentications, a package dedicated to it would be nice, but for now I guess we can move it to util.